src/d3a/models/market/one_sided.py

```diff
@@ -117,7 +117,7 @@ def accept_offer(self, offer_or_id: Union[str, Offer], buyer: str, *, energy: in
                     if trade_rate is None:
                         trade_rate = offer.price / offer.energy
 
-                    assert trade_rate >= math.floor(offer.price / offer.energy)
+                    assert trade_rate + 0.001 >= math.floor(offer.price / offer.energy)
```

Comment:
@spyrostz another point were you can input your FLOATING_POINT_TOLERANCE

Reply:
Indeed, once merged I will replace the the hardcoded value. Thanks!